virtual_ecosystem/models/abiotic/microclimate.py

```diff
@@ -381,7 +382,7 @@ def run_microclimate(
             ),
             dry_air_factor=abiotic_constants.dry_air_factor,
             cell_area=cell_area,
-            time_interval=1.0,  # TODO core_constants.seconds_to_hour,
+            time_interval=core_constants.seconds_to_hour,
```

Comment:
What does using the time_interval as `seconds_to_hour` mean, does it mean you are always running it for 3600s?

Reply:
yes, the update interval should always be 3600 seconds independent of the VE time interval. There is a conversion before the loop starts